### PR TITLE
Improve Hilla SEO

### PR DESCRIPTION
**Pull request by Mikaelsu**
Some optimization prompted by our SEO agency.

### CODE REVIEW

File: articles/lit/components/index.adoc [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Components
+title: Lit Web Components
 description: Hilla helps you build professional-looking UIs faster by including more than 45 UI components, provided by Vaadin, ranging from text fields to data grids.
 order: 30
 section-nav: flat expanded
 ---
 
-= Components
+= Lit Web Components
```
Comment (jouni) [inline articles/lit/components/index.adoc:8]: Do we really need to update this text as well? Did the SEO agency explicitly want us to change the `<h1>` as well, not just the “meta title”?

File: articles/lit/components/index.adoc [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Components
+title: Lit Web Components
```
[the remark this answers is not in the document]
Comment (jouni) [reply articles/lit/components/index.adoc:2]: This is inconsistent with the React counterpart. But I assume that is intentional, that people search for “Lit web components” more than “Lit UI component library”?

File: articles/react/components/index.adoc [hunk 3]
```diff
@@ -1,12 +1,15 @@
 ---
-title: Components
+title: React UI Component Library
+description: Hilla is a framework for building reactive web applications on Java backends. It seamlessly integrates a React TypeScript frontend with a Spring Boot backend using type-safe server communication, integrated tooling, and accessible UI components. Scroll to explore Hilla’s React component library.
 order: 40
 section-nav: expanded flat
 ---
 
-= Components
+= React UI Components
```
Comment (jouni) [inline articles/react/components/index.adoc:8]: Same comment here, that do we really need to change the `<h1>` in addition to the `<title>`?

File: articles/lit/components/index.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mikaelsu) [reply articles/lit/components/index.adoc:2]: Intentional, yes, and also true that I meant to change the `<title>` element. Thanks!

File: articles/react/components/index.adoc [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Components
+page-title: React UI Component Library
```
Comment (jouni) [inline articles/react/components/index.adoc:3]: I wonder if we should still add “Hilla” explicitly in the title as well? When the `page-title` frontmatter atteribute is used, it overrides the default generated title, and the value of the attribute is used directly (not appended/prepended with anything else).

```suggestion
page-title: React UI Component Library | Hilla Docs
```

File: articles/lit/components/index.adoc [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Components
+page-title: Lit Web Components
```
Comment (jouni) [inline articles/lit/components/index.adoc:3]:
```suggestion
page-title: Lit Web Components | Hilla Docs
```

File: articles/react/components/index.adoc (hunk 4 above)
Comment (Mikaelsu) [reply articles/react/components/index.adoc:3]: Good call!

**Pull request merged**